### PR TITLE
Terraform resources for the application

### PR DESCRIPTION
**Pull request by ev1lm0nk3y**
There are a couple of helm chart changes that will go nicely with another PR to be opened shortly.

### CODE REVIEW

File: deployment/gcp/sql.tf [hunk 1]
```diff
@@ -0,0 +1,83 @@
+locals {
+  db_admins = [
+    "evilmonkey@gmail.com",
+    "matt@fnf.org",
+  ]
+}
+
+resource "google_sql_database_instance" "ticket_booth" {
+  name             = "fnf-apps-db-instance"
+  database_version = "POSTGRES_15"
```
Comment (beingmattlevy) [inline deployment/gcp/sql.tf:10]: Should be ver 16

File: deployment/gcp/sql.tf [hunk 2]
```diff
@@ -0,0 +1,83 @@
+locals {
+  db_admins = [
```
Comment (beingmattlevy) [inline deployment/gcp/sql.tf:2]: Should this be a IAM GCP Group? Or are we good with individual admins for now?

**Review by beingmattlevy — COMMENTED** [review]
Couple of questions and one update needed.

File: deployment/gcp/sql.tf (hunk 1 above)
Comment (ev1lm0nk3y) [reply deployment/gcp/sql.tf:10]: done

File: deployment/gcp/sql.tf (hunk 2 above)
Comment (ev1lm0nk3y) [reply deployment/gcp/sql.tf:2]: GCP "groups" require a Google Worklspace. I wouldn't normally have the breakglass account either, but it seems prudent. Should I remove those "admin" individual users?

**Review by mike-matera — APPROVED** [review]
Nice! At some point we could add some of the other things to the TF... like the Kubernetes cluster and associated GCP service accounts.

**Pull request merged**